packages/edit-site/src/components/dataviews/dataviews.js

```diff
@@ -28,18 +28,63 @@ import TextFilter from './text-filter';
 export default function DataViews( {
 	data,
 	fields,
+	view,
+	onChangeView,
 	isLoading,
 	paginationInfo,
-	options,
+	options: { pageCount },
 } ) {
 	const dataView = useReactTable( {
 		data,
 		columns: fields,
-		...options,
+		manualSorting: true,
+		manualFiltering: true,
+		manualPagination: true,
+		enableRowSelection: true,
+		state: {
+			sorting: [
+				{ id: view.sort.field, desc: view.sort.direction === 'desc' },
+			],
+			globalFilter: view.search,
+			pagination: {
+				pageIndex: view.page,
+				pageSize: view.perPage,
+			},
+		},
+		onSortingChange: ( sortingUpdater ) => {
+			onChangeView( ( currentView ) => {
+				const [ { id, desc } ] =
+					typeof sortingUpdater === 'function'
```

Comment:
When we reset the sorting state, it becomes an empty array, so this breaks. I'm wondering if we should add a `default sorting` in the view config for that.

Reply:
How do we reset the sorting, so I can test?

Reply:
Ok found it and added support for "no sorting" to fix it.